### PR TITLE
[BugFix] Restricted y axis range in SBitThresh Summary Graph

### PR DESCRIPTION
**Pull request by bregnery**
## Description
The y-axis lost it's restriction in a previous version, I have fixed it so that the y-axis is fixed between 1e-1 and 1e8.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context
This was causing issues for the qc7 team, the y-axis should have remained fixed.

## How Has This Been Tested?
```bash
#!/usr/bin/env bash

#=========================================================================================
# Testing Suite for the Inflection Point finder ------------------------------------------
#=========================================================================================


echo "Welcome to Brendan's Testing Suite for the Inflection Point Finder"
echo "------------------------------------------------------------------------------------"
echo "This script executes a bunch of files known to cause issues for the inflection"
echo "point finder"
echo "------------------------------------------------------------------------------------"

echo "Did you remember to update??? With make pip and pip install rpm*.tar.gz?"

echo "File 1"
echo "This tests how the code handles a vfat with a bad vfat"
echo "Please make sure (2,5,4) is filled in your system_specfic_constants.py file"

# This file causes issues because of a zero slope issue
anaSBitThresh.py /data/bigdisk/GEM-Data-Taking/GE11_QC8//sbitRate/channelOR/2019.10.25.13.59/SBitRateData.root

echo "File 2"
echo "This tests how the code handles a detector with missing vfats"
echo "Please make sure (1,2,0) is filled in your system_specfic_constants.py file"

# This file is from the coffin where the detector is missing vfats
anaSBitThresh.py /data/bigdisk/GEM-Data-Taking/GE11_Integration//sbitRate/channelOR//2019.10.24.17.45/SBitRateData.root
```

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jsturdy — DISMISSED** [review]

File: utils/threshAlgos.py [hunk 1]
```diff
@@ -1508,16 +1508,14 @@ def getDetName(entry):
...
+                    # make sure the yaxis range is restricted
+                    graph.GetYaxis().SetRangeUser(1e-1,1e8)
 
                     # Draw a line on the graphs
-                    kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 10.0, dict_dacInflectPts[dacName][ohKey][vfat][0], ymax) )
+                    kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 1e-1, dict_dacInflectPts[dacName][ohKey][vfat][0], 1e8) )
                     kneeLine[vfat].SetLineColor(2)
                     kneeLine[vfat].SetVertical()
-                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )
+                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )
```
Comment (jsturdy) [inline utils/threshAlgos.py]: got some extra whitespace here

File: utils/threshAlgos.py [hunk 2]
```diff
@@ -1508,16 +1508,14 @@ def getDetName(entry):
                     if type(graph) == r.TH1F :
                         graph = r.TGraph(graph)
 
-                    # get maximum y value
-                    y = graph.GetY()
-                    y = np.array(y)
-                    ymax = np.amax(y)
+                    # make sure the yaxis range is restricted
```
Comment (jsturdy) [inline utils/threshAlgos.py]: comment probably not necessary

File: utils/threshAlgos.py [hunk 3]
```diff
@@ -1496,7 +1496,7 @@ def getDetName(entry):
                  # make nVFATs TLines
                 kneeLine= []
                 for vfat in range(0,nVFATS):
-                    canv_Summary1D.cd(vfat + 1).SetLogy()
+                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] ).SetLogy() # get pad corresponding to vfat
```
Comment (jsturdy) [inline utils/threshAlgos.py]: * extra whitespace in the function call
* comment probably not necessary

**Review by jsturdy — CHANGES_REQUESTED** [review]

File: utils/threshAlgos.py [hunk 4]
```diff
@@ -1508,14 +1508,13 @@ def getDetName(entry):
...
 
-                    # make sure the yaxis range is restricted
                     graph.GetYaxis().SetRangeUser(1e-1,1e8)
 
                     # Draw a line on the graphs
                     kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 1e-1, dict_dacInflectPts[dacName][ohKey][vfat][0], 1e8) )
                     kneeLine[vfat].SetLineColor(2)
                     kneeLine[vfat].SetVertical()
-                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )   
+                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )
```
Comment (jsturdy) [inline utils/threshAlgos.py]: still extraneous whitespace inside the function call (actually, also on L1514, now that I notice)

Comment (bregnery) [reply utils/threshAlgos.py]: Sorry, my style versus repo style. I fixed it.

File: utils/threshAlgos.py [hunk 5]
```diff
@@ -1508,13 +1508,10 @@ def getDetName(entry):
...
 
-                    # get maximum y value
-                    y = graph.GetY()
-                    y = np.array(y)
-                    ymax = np.amax(y)
+                    graph.GetYaxis().SetRangeUser(1e-1,1e8)
 
                     # Draw a line on the graphs
-                    kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 10.0, dict_dacInflectPts[dacName][ohKey][vfat][0], ymax) )
+                    kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 1e-1, dict_dacInflectPts[dacName][ohKey][vfat][0], 1e8) )
```
Comment (jsturdy) [inline utils/threshAlgos.py]: still extraneous whitespace here

**Review by jsturdy — APPROVED** [review]
ready for squash

**Pull request merged**